subprojects/docs/src/docs/userguide/migrating_from_maven.adoc

```diff
@@ -0,0 +1,518 @@
+
+Gradle has a similar system of project properties, although it differentiates between those and system properties.
+You can, for example, define properties in:
+
+* the build file
+* a `gradle.properties` file in the root project directory
+* a `gradle.properties` file in the `$HOME/.gradle` directory
+
+Those aren't the only options, so if you are interested in finding out more about how and where you can define properties, check out the <<build_environment#build_environment,Build Environment>> chapter.
+
+One important piece of behavior you need to be aware of is what happens when the same property is defined in both the build file and one of the external properties files: the build file value takes precedence.
+Always.
+Fortunately, you can mimic the concept of profiles to provide overridable default values.
+
+Which brings us on to Maven profiles.
+These are a way to enable and disable different configurations based on environment, target platform, or any other similar factor.
+Logically, they are nothing more than limited ‘if' statements.
+And since Gradle has much more powerful ways to declare conditions, it does not need to have formal support for profiles (except in the POMs of dependencies).
+You can easily get the same behavior by combining conditions with secondary build files, as you'll see next.
+
+Let's say you have different deployment settings depending on environment: local development (the default), a test environment, and production.
+To add profile-like behavior, first create build files for each environment in the project root: `profile-default.gradle`, `profile-test.gradle`, and `profile-prod.gradle`.
+Next, add a condition similar to the following to the main build file:
+
+[source,groovy]
+----
+if (!hasProperty('buildProfile')) ext.buildProfile = 'default'
+apply from: "profile-${buildProfile}.gradle"
+----
+
+All you have to do then is put the environment-specific configuration, such as project properties, dependencies, etc., in the corresponding build file.
+To activate a particular profile, you can just pass in the relevant project property on the command line:
+
+[listing.terminal]
+----
+$ gradle -PbuildProfile=test build
+----
+
+Or you can set the project property another way.
+It's up to you.
+And those conditions don't just have to check project properties.
+You could check environment variables, the JDK version, the OS the build is running on, and anything else you can imagine.
+
+One thing to bear in mind is that high level ‘if' statements make builds harder to understand and maintain, similar to the way they complicate Object-Oriented code.
+The same applies to profiles.
+Gradle offers you many better ways to avoid the extensive use of profiles that Maven often requires, for example by offering variants.
+
+For a lengthier discussion on working with Maven profiles in Gradle, look no further than http://gradle.org/feature-spotlight-gradles-support-maven-pom-profiles[this article] by Benjamin Muschko.
```

Comment:
moved to https://blog.gradle.org/maven-pom-profiles — also we can probably just say "by the Gradle team" if we must have a "by XYZ" at all.